integration_test/third_party_apps_data/applications/activemq/expected_metrics.yaml

```diff
@@ -0,0 +1,72 @@
+- type: workload.googleapis.com/activemq.connection.count
```

Comment:
Yep, it's called "type", because "name" was taken [1].

[1] https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.metricDescriptors#MetricDescriptor.FIELDS.type